test/geometry_tests.cpp

```diff
@@ -296,6 +298,89 @@ TEST(raytrace, negative_length) {
 	EXPECT_EQ(pixels.size(), 0);	
 }
 
+struct PolygonScenario{
+	std::vector<Cell> vertices;
+	int size_x;
+	int expected_size;
+	std::vector<Cell> expected_elements;
+};
+
+class polygonOutlineCellsTests : public :: testing::TestWithParam<PolygonScenario> {};
```

Comment:
Until we decide otherwise, let's hold the convention of classes starting with capitals

Reply:
Changed